source/amazon/services/supported-services/ecr-image-scanning.rst

```diff
@@ -29,6 +29,149 @@ AWS configuration
 
 AWS provides a `template <https://github.com/aws-samples/ecr-image-scan-findings-logger/blob/main/Template-ECR-SFL.yml>`_ for creating a stack in CloudFormation that loads the image scan findings from Amazon ECR in CloudWatch using an AWS Lambda function.
 
+Policy configuration
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think we can remove this header and add the new paragraphs to the previous section. Maybe something like this:

> AWS provides a template for creating a stack in CloudFormation that loads the image scan findings from Amazon ECR in CloudWatch using an AWS Lambda function. To be able to use this template it is necessary to create a custom policy granting the necessary permissions. To create a policy using the Amazon Web Services console, follow the AWS documentation.

This is just an example. Update it accordingly.